app/request_parsers/mouse_event.py

```diff
@@ -48,6 +51,8 @@ def parse_mouse_event(message):
         buttons=_parse_button_state(message['buttons']),
         relative_x=_parse_relative_position(message['relativeX']),
         relative_y=_parse_relative_position(message['relativeY']),
+        vwheel=message['vwheel'],
```

Comment:
I think we need to process the wheel values a little bit. On my system, the values are either 133 or 266, which I believe is pixels. We need to scale it somehow. It doesn't have to be perfect, but maybe it should just be divided by 3 to fit into -127 to 127.

Also, on my system, the scrolling is going in reverse. I think we need to negate it because a positive wheel value in JavaScript means the wheel scrolled down, whereas I believe HID treats positive as wheel scrolling up.